### ISSUE TITLE
Wrong release dates

### ISSUE THREAD

**Issue by Snowiiii** [GoldenStack/dayssincelastrustmcserver#3]
Hey, recently discovered this projects, looks nice :+1: 

But the release dates are wrong, I also wonder why you get your Informations from indirect sources, just look at the First commit. For Example

Pumpkin was released on on Jul 28 2024, First commit https://github.com/Snowiiii/Pumpkin/commit/419513d2e7b5118aa1c75fc87079e9dfc670dd9a

FerrumC was released on May 15 2024 https://github.com/ferrumc-rs/ferrumc/commit/f864bc4fbc7d0994c654330dc774f15832a5dd53

Basically all of the provided dates are wrong

**Comment by GoldenStack** [comment]
Hi,

I conflated "release" and "announcement" for the sake of wording. If you check [the source](https://github.com/GoldenStack/dayssincelastrustmcserver/blob/main/src/pages/index.astro#L45-L81) you can see I referenced my sources; in the case of Pumpkin and FerrumC, it was the first post I could find that seems to be announcing the project. I didn't consider the first commit because that isn't very well correlated with the actual "release" date, especially in cases like Graphite where the first commit was just the license.

**Comment by Snowiiii** [comment]
Okay, Why you decided to choose an announcement over an initial development release date of an project?

**Comment by GoldenStack** [comment]
I made this project to point out the hype that Rust Minecraft servers get, as well as how many there are, so when choosing dates I tried to choose the type of "announcement" that was more closely related to hype, not to when it necessarily became usable.

If you want I can update the date for Pumpkin and try DMing Sweattypalms to see if I should update FerrumC too.

**Comment by GoldenStack** [comment]
Addendum: this also acts as a barrier to make sure I really only add the more developed projects, since there are _many_ more implementations that are less complete and were never "announced".

**Comment by Snowiiii** [comment]
Nah, I was just interested why, At the end its your project of course you can choose which dates you want. I would may recommend to put it somewhere in the site so people actually know, because most of them will think that these are release dates. However, Good luck finding every puzzle in the World wide web, Many people like my own also made announcements on certain Discords. This should be the first announcement ever made, https://www.reddit.com/r/rust/comments/1egx979/minecraft_server_written_in_rust/ weird you didn't find it, its on the rust subreddit like FerrumC. 

As an "barrier" you could may consider a certain amounts of star's a repo needs to have

**Comment by GoldenStack** [comment]
Not finding your first announcement was my fault, oops. I just updated the date to the date of that post.

Thanks for letting me know! I'll close this issue now, LMK if there are any more concerns.
